content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -200,11 +209,42 @@ spec:
 You may also be able to mutate the incoming Pod, at admission time, to unset
 the `.spec.nodeName` field and to use a node selector instead.
 
+## Admin access
+
+{{< feature-state feature_gate_name="DRAAdminAccess" >}}
+
```

Comment:
```suggestion

You can mark a ResourceClaimTemplate as providing privileged features.

```

Reply:
Added, with one small tweak: "... mark` a *request* in a ResourceClaim *or* ResourceClaimTemplate ... as *having* ...".

It's per-request, not per claim (I wasn't precise there myself earlier, fixed), and it's not just at runtime ("providing") that such a request is special. It's also during scheduling.